tests/installation/partitioning_encrypted_no_lvm.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use strict;
+use warnings FATAL => 'all';
+use testapi;
+use parent "installbasetest";
+
+sub run {
+    my $self = shift;
+    my $partitioner = $testapi::distri->get_partitioner();
+
+    $partitioner->encrypt_partition_with_too_simple_password(is_lvm => 0);
```

Comment:
Also too long name for the function, I guess more OOP oriented would be to call it `encrypt_partition` and pass password and another param for example `complexity -> simple` or use a number and some comments for possible values.